RevenueCatUI/PaywallView.swift

```diff
@@ -3,6 +3,8 @@ import SwiftUI
 
 /// A full-screen SwiftUI view for displaying a `PaywallData` for an `Offering`.
 @available(iOS 16.0, macOS 13.0, tvOS 16.0, *)
```

Comment:
This version is still required, otherwise the compiler throws compatibility errors with APIs used inside 🤷🏻